### DISCUSSION TITLE
[Enhancement] Add Content Restriction Error Handler

### DISCUSSION THREAD

**Discussion by Sillyvan** [ciderapp/Cider#678] [Feature Request]
If you have never used Apple Music before you probably have the Content Restriction turned ON. 
This doesn't allow you to play songs that are tagged as explicit.

Cider doesn't know about this and just wont play songs.

how the error shows on the web client:
![image](https://user-images.githubusercontent.com/38138071/148106695-23695422-ef41-4659-b368-932e3da2b80f.png)

**Comment by iammarxg** [comment]
I get this too, it's an issue on the music.apple.com website. It won't allow you to change the rating unlike iTunes on Windows, and Apple Music on iOS and Android.

**Comment by Sillyvan** [comment]
yep. its probably still possible. if you check what request itunes or the app makes over the network. it might be possible

**Comment by iammarxg** [comment]
Oh yeah, I forgot to mention that Apple Music has enabled explicit content in my region on their website and this problem was solved for me.

**Comment by SkyRexHUN** [comment]
I'm having the same problem, music just stops playing when an explicit song comes up, and the whole player becomes unuseable until I restart Cider.
